snapcraft/cli/lifecycle.py

```diff
@@ -137,6 +140,24 @@ def snap(directory, output, **kwargs):
 
 
 @lifecyclecli.command()
+@click.argument('directory')
+@click.option('--output', '-o', help='path to the resulting snap.')
+def assemble(directory, output, **kwargs):
+    """Assemble a snap from a directory layout.
+
+    The layout of <directory> should contain a valid meta/snap.yaml
```

Comment:
How about mentioning `prime` here?

Reply:
this is a no, `prime` could be a foreign concept to the assembly. Any other wording is welcome

Reply:
Not sure what you mean by foreign concept... but maybe more explicit is better. How about

`[...], optionally meta/gui and meta/manifest.yaml
as well as any other files making up the filesystem of the snap image.`

Reply:
foreign as in a user that has no idea will just suffer with those concepts. I would rather say _The target <directory> should contain a valid snap layout, which would minimally have a valid meta/snap.yaml_ (and I would remove what follows the comma).

I am not for listing only *some* optional entries and folks should be introduced to this as they go, much less for this assemble command which is already an advanced command.

@evandandrea your thoughts would be appreciated.

Reply:
`meta/snap.yaml` is all that's required to make something a snap, whether the directory was created with `snapcraft prime` or not. I think the description is fine as-is.